.github/workflows/nightly.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
           S3_BUILD_CACHE_SECRET_KEY: ${{ secrets.S3_BUILD_CACHE_SECRET_KEY }}
         with:
           command: ./gradlew testJava${{ matrix.java }} --stacktrace -Dorg.gradle.java.installations.paths=${{ steps.setup-test-java.outputs.path }}
-          timeout_minutes: 90
+          timeout_minutes: 180
```

Comment:
That's long 😮

Reply:
That's VERY long. I hope this helps me to get a full build scan which may reveal something. I have no idea why `testJava15` takes so much time.